lib/plugins/aws/package/lib/generateCoreTemplate.js

```diff
@@ -25,6 +25,14 @@ module.exports = {
 
     const bucketName = this.serverless.service.provider.deploymentBucket;
     const isS3TransferAccelerationEnabled = this.provider.isS3TransferAccelerationEnabled();
+    const isS3TransferAccelerationDisabled = this.provider.isS3TransferAccelerationDisabled();
+
+    if (isS3TransferAccelerationEnabled && isS3TransferAccelerationDisabled) {
+      const errorMessage = [
+        'You cannot enable and disable S3 Transfer Acceleration at the same time',
+      ].join('');
+      throw new this.serverless.classes.Error(errorMessage);
```

Comment:
To introduce promise consistency over time, we should not throw here but reject.

```
return BbPromise.reject(new this.serverless.classes.Error(errorMessage));
```

Reply:
Done :)